hathor/exception.py

```diff
@@ -18,6 +18,11 @@ class HathorError(Exception):
     pass
 
 
+class BuilderError(Exception):
+    """Base class for builder."""
```

Comment:
Maybe update to "Base class for exceptions in the Builder"?

Reply:
Done! Thanks!